code/datums/proximity_monitor/proximity_monitor.dm

```diff
@@ -13,6 +13,8 @@
 		COMSIG_ATOM_EXITED = PROC_REF(on_uncrossed),
 		COMSIG_ATOM_INITIALIZED_ON = PROC_REF(on_initialized),
 	)
+	///A list of all mobs within this radius, to track what signals are already registered
+	var/list/mobs_in_range
```

Comment:
not sure why this code is on the parent, and this is not harddel safe, and I dont really get what its doing AND theres no docs